requirements.txt

```diff
@@ -16,6 +16,7 @@ click-plugins==1.1.1
 click-repl==0.2.0
 cryptography==38.0.1
 diskcache==5.4.0
+dulwich==0.20.46
```

Comment:
```suggestion
```
This is a development-only dependency (does not belong here, but only in the `.dev/requirements.txt` file).